src/pages/experimental/import-pattern/index.mdx

```diff
@@ -17,18 +17,23 @@ The import action transfers data or objects from an external source into a syste
 
 [Vikki Paterson](https://github.com/vikkipaterson)
 
-![Example of an import modal in context](/images/0.5.png)
+![Example of an import modal](/images/0.png)
 
-<Caption>Example of an import modal in context</Caption>
+<Caption>Example of an import modal</Caption>
+
+## Choose a file (modal)
+
+Where a single file is imported, import can take place in a modal.
 
-## Choose a file
+When importing users should have the option of uploading a local file using the file upload component, offering file drag and drop and the ability to browse locally to select a file.  Additionally provide the ability to add a file from an external resource.
 
-Wherever possible there should be a drag and drop zone. Use a file drop component and offer the option to browse locally to select a file. Note that file selection should be restricted to allowed file types.
+
+If additional actions (e.g. adding metadata) are required to be performed on the file(s) consider whether a it would be more appropriate to use a side panel or full page instead of a modal. 
```

Comment:
```suggestion
If additional actions (e.g. adding metadata) are required to be performed on the file(s), consider whether it would be more appropriate to use a side panel or full page instead of a modal. 
```